pkg/execute/action.go

```diff
@@ -0,0 +1,88 @@
+package execute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/kubeshop/botkube/pkg/config"
+)
+
+func (e *DefaultExecutor) runActionCommand(ctx context.Context, args []string, clusterName string) (string, error) {
+	if len(args) < 2 {
+		return "", errInvalidCommand
+	}
+
+	validSubCmd := false
+	for _, cmdName := range []string{"action", "actions", "act"} {
+		if args[1] == cmdName {
+			validSubCmd = true
+			break
+		}
+	}
+	if !validSubCmd {
+		return fmt.Sprintf("'%s' is not a valid subcommand", args[1]), nil
+	}
+
+	var cmdVerb = args[0]
+	defer func() {
+		cmdToReport := fmt.Sprintf("%s %s", args[1], cmdVerb)
```

Comment:
It's inverted - if you apply the previous snippet, then you can use:
```suggestion
		cmdToReport := fmt.Sprintf("%s %s", cmdVerb, cmdRes)
```